archive/v1/README.md

```
@@ -0,0 +1,230 @@
+- **PDFs:** `.pdf`
+
+## Prerequisites 💻
+
+- **Operating System:** Compatible with Windows, macOS, and Linux.
+- **Python Version:** Python 3.12
+- **Conda:** Anaconda or Miniconda installed.
+- **Git:** For cloning the repository (or you can download the code as a ZIP file).
+
+## Installation 🛠
+
+> For SDK installation and model-related issues, please post on [here](https://github.com/NexaAI/nexa-sdk/issues).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)